src/Form/ContentpoolReplicationFilterForm.php

```diff
@@ -209,7 +210,7 @@ protected function fetchTermReferenceFilter(RemoteInterface $remote) {
         '#description' => $this->t('Term reference filter: only sync content with selected %term.', ['%term' => $field_data['label']]),
         '#attached' => [
           'library' => [
-            'contentpool_client/treeselect_filter',
+            'contentpool_client/replication_filter_form',
```

Comment:
oops, sorry couldn't test after i renamed the lib, i had the permission error again when fetching from contentpool.